### PR TITLE
docs(cli): add `api` to override sample command

### PR DESCRIPTION
**Pull request by josefaidt**
_Issue #, if available:_

n/a

_Description of changes:_

adds `api` to sample override command `<api|auth|project|storage>`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/pages/cli/migration/override.mdx [hunk 1]
```diff
@@ -1,16 +1,16 @@
 export const meta = {
   title: `Migration to enabled override feature`,
-  description: `Upgrading to Amplify CLI version 7 and above with a project created prior requires an migration to enable the new "override" capability.`,
+  description: `Upgrading to Amplify CLI version 7 and above with a project created prior requires an migration to enable the new "override" capability.`
```
Comment (sachscode) [inline src/pages/cli/migration/override.mdx:3]: should be "requires a migration" ?

**Review by sachscode — COMMENTED** [review]
LGTM, one question

File: src/pages/cli/migration/override.mdx (hunk 1 above)
Comment (josefaidt) [reply src/pages/cli/migration/override.mdx:3]: you are absolutely correct I did not notice that

Comment (sachscode) [reply src/pages/cli/migration/override.mdx]: Thanks!

**Review by siegerts — APPROVED** [review]

**Pull request merged**